builder-support/debian/authoritative/debian-buster/config/gmysql.conf

```diff
@@ -0,0 +1,68 @@
+# See https://doc.powerdns.com/authoritative/backends/generic-mysql.html
+launch+=gmysql
+
+#################################
+# gmysql-dbname	Database name to connect to
+#
+# gmysql-dbname=powerdns
+gmysql-dbname=_DBNAME_
```

Comment:
this smells of db-config

Reply:
it was, but right now its used as the sed template for the autopkgtest. Somehow having separate files (one example file, one autopkgtest file) might be cleaner, but its what we have today.